tests/e2e/specs/factory/Factory.spec.ts

```diff
@@ -88,6 +88,13 @@ suite(
 			});
 		}
 
+		if (BASE_TEST_CONSTANTS.IS_RESTART_EXISTING_WORKSPACE) {
+			test('Restart existing workspace', async function (): Promise<void> {
```

Comment:
'Restart existing workspace' -> `Select opening existing workspace instead of creation new one`

Reply:
Fixed.

Reply:
thanks